### DISCUSSION TITLE
DeskThing Alpha Release 3.0

### DISCUSSION THREAD

**Discussion by ItsRiprod** [ItsRiprod/DeskThing#2] [Announcements]
# Desk Thing Alpha 2

This is the last and final stable release of The Desk Thing that uses the command terminal and .env files to manage loaded apps and API keys.

The current functionality is somewhat limited but completely functional

You can still DM (@ItsRiprod) on Discord but I will recommend using the latest version. If there are any features of this that you would like to see in the latest version, I will most likely be able to add it to the latest release instead of supporting this one.

## What's Changed
* Added the ability for apps to be modular
* Migrated to Vite from CRA

## New Contributors
* @ItsRiprod made their first contribution in https://github.com/ItsRiprod/DeskThing/pull/1

**Full Changelog**: https://github.com/ItsRiprod/DeskThing/compare/v0.2.0-alpha...v0.3.0-alpha

<hr /><em>This discussion was created from the release <a href='https://github.com/ItsRiprod/DeskThing/releases/tag/v0.3.0-alpha'>DeskThing Alpha Release 3.0</a>.</em>

**Comment by chaseadam** [comment]
tag is 3.0 but titled 2.0 release in github releases

**Reply by ItsRiprod** [reply]
Thank you,
Is now fixed for both this version and last
